mautic/mautic
# Google Analytics - IP Anonymization

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5483
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. ) 

[//]: # ( Required: )
#### Description:
Just added new option to  anonymize the IP address for some country law restrictions. More info https://developers.google.com/analytics/devguides/collection/analyticsjs/ip-anonymization

![image](https://user-images.githubusercontent.com/462477/34611933-ed447ff0-f227-11e7-8348-2e5c84e077ad.png)

#### Steps to test this PR:
1.  Apply PR and setup Google Analytics in Configuration > Tracking Settings tab
2.  Enable IP Anonymization and Enable on Mautic landing page
3. Go to Mautic landing page and see source if  this line was added
`ga('set', 'anonymizeIp', true);`
